ui/app/controllers/exec.js

```diff
@@ -17,7 +17,10 @@ export default class ExecController extends Controller {
   @service system;
   @service token;
 
-  queryParams = ['allocation'];
+  queryParams = [
+    'allocation',
+    'namespace'
```

Comment:
`thought`: `namespace` is a reserved keyword in Ember. We normally alias to `qpNamespace` as a result. However, since we have no logic that depends on this variable, we should be fine. But worth noting.